go-controller/pkg/ovn/policy.go

```diff
@@ -1741,3 +1537,251 @@ func PortGroupHasPorts(nbClient libovsdbclient.Client, pgName string, portUUIDs
 
 	return sets.NewString(pg.Ports...).HasAll(portUUIDs...)
 }
+
+type sharedPortGroupInfo struct {
```

Comment:
maybe netpolLocalPodsPortGroup gives a bit more info about how exactly this port group is used?